api/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "prepare": "husky install",
+    "prepare": "cd .. && husky install test/.husky",
```

Comment:
nit: Could we undo this? Should just be ran in the command line as:
`cd {dir} && npm rum prepare` or `cd {dir} && husky install`